logic/account.js

```diff
@@ -508,25 +510,18 @@ class Account {
 						}
 
 						// If updated value is positive number
-						if (value.greaterThan(0)) {
+						if (value.isGreaterThan(0)) {
 							promises.push(
-								dbTx.accounts.increment(
-									address,
-									updatedField,
-									value.floor().toString()
-								)
+								dbTx.accounts.increment(address, updatedField, value.toString())
```

Comment:
why did we remove `floor` here? are you sure that it will not have any implications? a good suggestion from my side for this PR, in general, is to try to sync the `testnet` from scratch, because in the previous encounter related to bignum changes we faced lot of issues, so it will be great if you can sync using this branch.

Reply:
@4miners was in the loop. We confirmed that `floor()` wasn't adding anything in those cases.

Reply:
ok, if you guys have a clue that it will work then cool, my only suggestion was a quick check to verify if these big number changes work, if you are confident then 👍